src/Data/Foldable/Compat.hs

```diff
@@ -2,14 +2,37 @@
 module Data.Foldable.Compat (
   module Base
```

Comment:
Same, here. Only re-exporting `Base` for the latest version of `base` allows us to use the `.Compat` module with the latest version of `base` as reference.